lib/aws/log_bucket.ts

```diff
@@ -0,0 +1,135 @@
+                        cpu: "1",
+                        memory: "1200Mi"
+                    },
+                    requests: {
+                        cpu: "200m",
+                        memory: "150Mi"
+                    }
+                },
+                rbac: {
+                    create: false
+                 },
+                livenessProbe: null,
+                readinessProbe: null,
+                service: {
+                    enabled: false,
+                    type: "ClusterIP",
+                 },
+                image: {
+                    repository: "fluent/fluentd-kubernetes-daemonset",
+                    pullPolicy: "IfNotPresent",
+                    tag: "v1.16-debian-s3-1"
+                },
+                env: [
+                    {
+                        name: "S3_BUCKET",
+                        value: this.bucket.bucketName,
+                    },
+                    {
+                        name: "S3_REGION",
+                        value: process.env.CDK_DEFAULT_REGION,
+                    }
+
+                ],
+                terminationGracePeriodSeconds: 30,
+                dnsPolicy: "ClusterFirst",
+                restartPolicy: "Always",
+                schedulerName: "default-scheduler",
+                securityContext: {},
+                fileConfigs:{
+                    "01_sources.conf":` <source>
+                    @type tail
+                    @id in_tail_hyperswitch-server-router_logs
+              
+                    path /var/log/containers/hyperswitch-*.log
+                    pos_file /var/log/fluentd-hyperswitch-server-router-containers.log.pos
+                    tag "hyperswitch.*"
+                    read_from_head true
+                    <parse>
+                      @type regexp
+                      expression /^(?<time>.+) (?<stream>stdout|stderr)( (?<logtag>.))? (?<log>.*)$/
+                    </parse>
+                  </source>`,
+                  "02_filters.conf":"",
+                  "03_dispatch.conf":"",
+                  "04_outputs.conf": `<match hyperswitch.**>
+                  <format>
+                    @type json
+                  </format>
+                  @type copy
+                  <store>
+                    @type stdout
+                  </store>
+                  <store>
+                    @type s3
+                    s3_bucket "#{ENV['S3_BUCKET']}"
+                    s3_region "#{ENV['S3_REGION']}"
+                    path "hyperswitch-logs/%Y/%m/%d/$\{tag\}/"
+                    <buffer tag,time>
+                      @type file
+                      path /var/log/fluent/s3
+                      timekey 3600 # 1 hour partition
+                      timekey_wait 10m
+                      timekey_zone +0530
+                      chunk_limit_size 256m
+                      flush_at_shutdown
+                    </buffer>
+                  </store>
+                </match>`
```

Comment:
Can we read this configuration from a file?

Reply:
sure, lets take the refactoring in separate pr!